changelog.d/1114.bugfix

```diff
@@ -0,0 +1,7 @@
+Correct nick validation
+
```

Comment:
Oh and you are more than welcome to append a "Fixed by @dalcde" to get credited btw :)